weave/weave_client.py

```diff
@@ -409,7 +413,16 @@ def create_call(
                 self._anonymous_ops[op] = _build_anonymous_op(op)
             op = self._anonymous_ops[op]
         if isinstance(op, Op):
-            op_def_ref = self._save_op(op)
+            if isinstance(op, partial):
```

Comment:
can this block be written as a function in op.py?

Reply:
clarified as `maybe_unbind_method` in `op.py`